tests/pipelines/tf-ep-dns-pull-requests.yaml

```diff
@@ -0,0 +1,159 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ 
+jobs:
+
+###
+# run-example-ep-dns-gce-acme-pr
+###
+- name: run-example-ep-dns-gce-acme-pr
+  serial: true
+  build_logs_to_retain: 30
+  plan:
+  - get: git
+    trigger: false
+  
+  - get: git-pull-requests
+    trigger: true
+    version: every
+  
+  - put: git-pull-requests
+    params:
+      path: git-pull-requests
+      status: pending
+
+  - task: run-example-upgrade
+    params:
+      git_src: git
+      git_target: git-pull-requests
+      service_account_json: ((pr-project.service_account_json))
+      project_id: ((pr-project.project_id))
+      backend_bucket: {{backend_bucket}}
+      backend_prefix: terraform-google-endpoints-dns/
+      env_name: tf-ci-ep-dns-gce-acme-pr
+      example_dir: examples/gce-acme
+      region: us-west1
+      zone: us-west1-b
+      TF_VAR_network_name: tf-ci-ep-dns-gce-acme-pr
+      TF_VAR_region: us-west1
+      TF_VAR_zone: us-west1-b
+      init_script: |
+        # Create tfvars file
```

Comment:
Couldn't these be `TF_VAR_` env variables instead/as well?

Reply:
Yes! Good call, I'll get that updated.